core/src/com/cloud/agent/api/StartupRoutingCommand.java

```diff
@@ -35,7 +35,7 @@
     long memory;
     long dom0MinMemory;
     boolean poolSync;
-
+    private boolean _supportsClonedVolumes;
```

Comment:
I agree with Rafael on this. Let's not introduce more members starting with underscore but grow to cleaner code (slowly)